www/js/project-create-open-handlers.js

```diff
@@ -196,9 +190,7 @@ function create(projectName, projectId, projDir) {
             var diff = complete - start;
```

Comment:
Since we are no longer logging the total project create time, we don't need to keep these variables around anymore. My bad, I forgot this when I removed logging.